UnitTests/PaymentsCoreTests/APIClient_Tests.swift

```diff
@@ -91,11 +115,14 @@ class APIClient_Tests: XCTestCase {
         mockURLSession.cannedURLResponse = nil
 
         apiClient.fetch(endpoint: fakeRequest) { result, _ in
-            guard case .failure(.invalidURLResponse) = result else {
+            switch result {
+            case .success(_):
                 XCTFail()
-                return
+            case .failure(let error):
+                XCTAssertEqual(error.domain, APIClientError.domain)
+                XCTAssertEqual(error.code, APIClientError.Code.invalidURLResponse.rawValue)
+                XCTAssertEqual(error.localizedDescription, "An error occured due to an invalid HTTP response. Contact developer.paypal.com/support.")
             }
-
```

Comment:
nit: it looks like in some places we include a space before `expect.fulfill()` and other places we've removed them. Should this be the same everywhere?